assets/js/components/adminbar/AdminBarImpressions.stories.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * Admin Bar Impressions Component Stories.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import { withWidgetComponentProps } from '../../googlesitekit/widgets/util';
+import { setupSearchConsoleMockReports, widgetDecorators } from './common.stories';
+import AdminBarImpressions from './AdminBarImpressions';
+import WithRegistrySetup from '../../../../tests/js/WithRegistrySetup';
+
+const WidgetWithComponentProps = withWidgetComponentProps( 'widget-slug' )( AdminBarImpressions );
+
+const Template = ( { setupRegistry, ...args } ) => (
+	<WithRegistrySetup func={ setupRegistry ? setupRegistry : () => {} }>
```

Comment:
Same as above/for all instances here.

Also, you could do:
```suggestion
const Template = ( { setupRegistry =  () => {}, ...args } ) => (
	<WithRegistrySetup func={ setupRegistry }>
```

Reply:
👍